packages/forma-36-react-components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -86,7 +86,7 @@ export interface DropdownProps {
   /**
    * Child nodes to be rendered in the component
    */
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
Is this correct?

I think we do want to make children mandatory in Dropdown
A person can't/shouldn't do this:

```
<Dropdown />
```

right?